client/client-multi/client-c/src/kaa/kaa_logging.c

```diff
@@ -324,10 +368,13 @@ kaa_error_t kaa_log_collector_create(kaa_log_collector_t **log_collector_p
     return KAA_ERR_NONE;
```

Comment:
Please, call `KAA_FREE` instead of `kaa_log_collector_destroy` at https://github.com/kaaproject/kaa/pull/633/files#diff-cfa7a72c56371fdb77ac97b867682665L319